src/pacasam/analysis/stats.py

```diff
@@ -36,12 +36,16 @@ def compare(self, df_database: DataFrame, df_sampling: DataFrame):
             if df_sampling[key].nunique() <= 1:
                 continue
             # Compare prevalence of all boolean descriptors.
-            comparison_df_by_key = self.compare_by_key(df_database, df_sampling, key, self.compare_bools)
+            comparison_df_by_key = self.compare_by_key(
```

Comment:
t'as changé les paramètres de ton linter par rapport aux versions précédentes ?
Le fait que tu changes ton formattage automatiquement rend les reviews un peu plus galères, si jamais il y a un moyen de rendre ça consistent sur un projet donné ce serait cool

Reply:
Je vais passer un coup avec le paramétrage de pyproject.toml. 
En gros le paramétrage de VS Code n'est pas forcément le même que celui du projet ; je vais voir si ça peut être le cas mais pas sûr. C'est là où pre-commit peut être pertinent... Voire, en Action github. Vous utilisez ça parfois au RDI ?

Reply:
Je vais essayer les deux approches proposées ici https://dev.to/adamlombard/vscode-setting-line-lengths-in-the-black-python-code-formatter-1g62

Reply:
OK j'ai compris ce qu'il se passe. J'avais coexistence entre le formatteur Black inclue dans l'extension VS Code "Python", et ce qui est destiné à la remplacer : [l'extension Black-Formatter](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter). En activant celle-ci, et en retirant les paramètre de nb max de ligne de l'extension Python, Black-Formatter va bien utiliser le paramétrage situé dans pyproject.toml.

Après merge des trois MR actuelles je passerai un coup de formattage, il y a moins d'une dizaine de fichiers concernés.